Makefile

```diff
@@ -253,6 +253,15 @@ wad-image-help: wad-image-common
 	@echo "      make wad-image-diff WI_VERBOSE=t WI_GIF=t WI_SHOW=t WI_CMD=animate \
 WI_COMMIT=\"0c004ce~..0c004ce\""
 
+# Test that WAD files have the expected map names.
+test-map-names:
+	tests/test-map-names
```

Comment:
This file belongs in the `scripts` directory

Reply:
I didn't write my explanation comment fast enough, which might of helped. How would you like it to look? Is that a good enough reason to have a new `tests` directory? If not and you still want it in `scripts` do you still want the `test` prefix for the file name and build targets?